### ISSUE TITLE
Landing Page for Event

### ISSUE THREAD

**Issue by praveenscience** [praveenscience/Creative-Simple-Landing-Pages#76]
Hey Open Sourcers, would love to get a landing page done here:

![image](https://user-images.githubusercontent.com/1830380/194890064-fb8c732a-312e-4059-aead-abb3293ea6ee.png)

The features should be:

- Very simple HTML & CSS!
- No JavaScript or minimal one please!
- Should be Responsive

Reference: [Live Product Demo | UXPin Merge](https://get.uxpin.com/live-demo-uxpin-merge/index.html)

Also see below for the full screenshots.

**Full design:**

Desktop Design:

![get uxpin com_live-demo-uxpin-merge_index html](https://user-images.githubusercontent.com/1830380/194890440-2039762f-f0ba-4db4-bf14-465672675575.png)

Mobile Design:

![get uxpin com_live-demo-uxpin-merge_index html](https://user-images.githubusercontent.com/1830380/194890475-fa4c78fe-9f46-4ccd-92f1-ecc9f8e7b879.png)

Thanks! 💪🏻

**Comment by AbhishekTiwari23** [comment]
I would love to work on this, but I need Figma design and photos. Thanks

**Comment by praveenscience** [comment]
@AbhishekTiwari23 The original page itself is linked in the issue. Can you check if you can do it? Thanks! 💪🏻 The page has already been there. I love the design, let's make it open sourced! 😅

**Comment by AbhishekTiwari23** [comment]
So I have to create the same page. Like That one?

**Comment by praveenscience** [comment]
@AbhishekTiwari23 Correct. But with minimal HTML & CSS only. Feel free to use the Bootstrap CDN or CDN of your choice. Minimalism is the key. Should I assign this to you? 💪🏻

**Comment by AbhishekTiwari23** [comment]
Yes, I am waiting

**Comment by praveenscience** [comment]
@AbhishekTiwari23 It's yours... When can I expect something? 😅

**Comment by AbhishekTiwari23** [comment]
Please give me a week at least.

**Comment by praveenscience** [comment]
It's gonna be a competition now.... Maybe both the projects win! 😄 Yes, @aadii0408!

**Comment by praveenscience** [comment]
@aadii0408 Get your version too! Would love to see both your versions! 💪🏻

**Comment by praveenscience** [comment]
The fun part is going to be how each of you are going to build it! 😄 So whoever is ready, please comment and get your version built! 💪🏻 Very eager to see all the versions! 👍🏻

**Comment by Deepthi-v-s** [comment]
@praveenscience Can I try for this please? 😊

**Comment by praveenscience** [comment]
@Deepthi-v-s Yay, looking forward...

**Comment by Subha-5** [comment]
@praveenscience I would like to give this a try, please assign me

**Comment by AbhishekTiwari23** [comment]
> Any updates on this @aadii0408 @AbhishekTiwari23?

Actually, My internet is not working correctly, and there is a fest going on in my college, so please give me some time.

**Comment by praveenscience** [comment]
@AbhishekTiwari23 Sure...
@Subha-5 Try it out. Looking forward...

**Comment by rohannsahh** [comment]
hey is this repo still open 
can you assign me plz?

**Comment by praveenscience** [comment]
Any updates? The assigned folks, please tell me?
@aadii0408
@AbhishekTiwari23
@Subha-5
@Deepthi-v-s
@rohannsahh

**Comment by Deepthi-v-s** [comment]
@praveenscience Almost done with the first trial, will submit a PR tomorrow! 🤩

**Comment by niralverma** [comment]
I wanna contribute in this. Could you please assign me too?

**Comment by praveenscience** [comment]
Any updates? It's the last few hours of Hacktoberfest? 😁
